puzzles/templates/modals/add_tag.html

```diff
@@ -0,0 +1,22 @@
+<div class="modal fade" id="addtag" tabindex="-1" role="dialog" aria-labelledby="addtag" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+
+        </div>
+    </div>
+</div>
+
+<script type="text/javascript">
+$(function () {
+  $(".addtag").click(function() {
+    var pk = $(this).data("pk");
+    $.get({
+        "url": `/puzzles/add_tags_form/${pk}`,
+        'csrfmiddlewaretoken': '{{ csrf_token }}'
```

Comment:
Is this necessary? I don't think I need that for set_metas, what is different here?

Reply:
Oh, guess this isn't necessary. Thanks for catching this. It was just something I added while trying to figure out how to fix a `A {% csrf_token %} was used in a template, but the context did not provide the value.` error